README.md

```diff
@@ -157,7 +157,7 @@ This build is optional, as MFEM can be build without METIS by specifying
 
 Clone and build the parallel version of MFEM:
 ```sh
-~> git clone git@github.com:mfem/mfem.git ./mfem
+~> git clone https://github.com/CEED/MFEM.git ./mfem
```

Comment:
The upstream repository (https://github.com/mfem/mfem) exists, but https is indeed a better protocol for this level of documentation (since it doesn't require the user to have a github account and have uploaded their SSH public key). Can you amend this PR to only change the protocol? Looks good otherwise.

The last argument on this command line is optional and could be removed.